src/components/tree-views-container.js

```diff
@@ -728,15 +730,33 @@ const TreeViewsContainer = () => {
                 error={renameError}
             />
             <DeleteDialog
-                message={intl.formatMessage({
-                    id: 'deleteDirectoryDialogMessage',
-                })}
+                message={
```

Comment:
must be managed inside the deleteDialog component not in tree-views-container

Reply:
done